plasmapy/physics/distribution.py

```diff
@@ -37,8 +37,7 @@ def Maxwellian_1D(v: units.m/units.s,
     Returns
     -------
     f : Quantity
-        probability in Velocity^-1, normized so that:
-        $\int_{- \infty}^{\infty} f(v) dv = 1}
+        probability in Velocity^-1, normized so that: :math:`\int_{-\infty}^{+\infty} f(v) dv = 1`
```

Comment:
Some minor errors in the original version of this line but #152 is working on that so don't worry about it :)